package/MDAnalysis/analysis/rms.py

```diff
@@ -265,34 +267,25 @@ def process_selection(select):
     return select
 
 
-class RMSD(object):
-    """Class to perform RMSD analysis on a trajectory.
-
-    Run the analysis with :meth:`RMSD.run`, which stores the results
-    in the array :attr:`RMSD.rmsd`::
```

Comment:
The information for what the results are should still be *somewhere* as well as an indication what algorithms are used.

What was the rationale for removing it from the class string?